frontend/src/utils/rules.js

```diff
@@ -7,6 +7,8 @@ const rules = {
         return true
       } else if (Array.isArray(v)) {
         return v.length > 0 || REQUIRED_MSG
+      } else if (typeof v === 'boolean') {
+        return v != null
```

Comment:
I think this would always return true?

Reply:
agreed. I'll move it to the 1st condition as below:
```
if (v === 0 || typeof v === 'boolean') {
        return true
      }
```

Reply:
it's updated!